environment.yml

```diff
@@ -10,7 +10,8 @@ dependencies:
   - openms::openms=2.7.0pre
   - openms::openms-thirdparty=2.7.0pre # 08.02.2021
   - bioconda::bioconductor-msstats=3.20.1 # will include R
-  - bioconda::sdrf-pipelines=0.0.13 # for SDRF conversion
+  - bioconda::sdrf-pipelines=0.0.14 # for SDRF conversion
+  - bioconda::thermorawfileparser=1.2.3
```

Comment:
TRFP: this should be done in the openms-thirdparty conda package. I will make sure to do it at the next update.